src/app/app.component.ts

```diff
@@ -1,10 +1,118 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Item } from './models/item';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'angular-green-grocers';
-}
+  groceries: Item[] = [
```

Comment:
It looks like you had an issue getting the data, but it might have been better to raise a ticket rather than hard-coding your data. You wouldn't be able to hard-code everything in a typical shop scenario, as there might be a lot of data, and the data is likely to change over the course of the shops life.